src/actions/actionDistribute.tsx

```diff
@@ -39,6 +39,7 @@ const distributeSelectedElements = (
 
 export const distributeHorizontally = register({
   name: "distributeHorizontally",
+  trackEvent: { category: "element" },
```

Comment:
pass `action: distributeHorizontally` ?

Reply:
`action` defaults to `action.name` so no need

Reply:
Yep realized that after full review 👍🏻